python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -151,7 +163,8 @@ def object_list_get(self, name):
         """
 
         if name not in self._object_list:
-            object_table = Simbad.query_object(name)
+
+            object_table = self._query_object(name)
 
             if object_table is None:
                 raise RuntimeError(f"Could not find {name} in Simbad database.")
```

Comment:
The `_query_object` function not only queries Simbad but also the local catalogs so this error message is not correct anymore.

Reply:
I moved this check to `_query_object`, expanding the error message if the catalog is loaded and the target is not found.